doc/api/errors.md

```diff
@@ -2358,6 +2358,14 @@ Accessing `Object.prototype.__proto__` has been forbidden using
 [`Object.setPrototypeOf`][] should be used to get and set the prototype of an
 object.
 
+<a id="ERR_READLINE_CLOSED"></a>
+
+### `ERR_READLINE_CLOSED`
```

Comment:
nit: I think it would be better if we rename this to something more generic like `ERR_USE_AFTER_CLOSE` and replace all the other error codes with this one in a separate PR. Refs: https://github.com/nodejs/node/pull/41660#issuecomment-1019425537.